Jenkinsfiles/hydra_copy

```diff
@@ -28,7 +41,7 @@ pipeline {
             }
             steps {
                 script {
-                    sh "python3 hydrascrape/hydrascrape.py hydra.vedenemo.dev ${params.project} ${params.jobset} handled_builds "+ '\'python3 hydrascrape/action.py\''
+                    sh "python3 hydrascrape/hydrascrape.py ${params.server}.vedenemo.dev ${params.project} ${params.jobset} handled_builds "+ '\'python3 hydrascrape/action.py\''

```

Comment:
Is params.server name or id of the server?

I guess this is an improvement (makes the script at least a bit more flexible) even if it's the id, and thus means that only servers accessible by <id>.vedenemo.dev are supported by current hydrascrape version.

Reply:
I do not know is that hydra part name or ID but yes it just uses xxxx.vedenemo.dev url's atm.